lib/ovs-router.c

```
@@ -529,23 +864,60 @@ scan_ipv4_route(const char *s, ovs_be32 *addr, unsigned int *plen)
+{
+    struct route_data rd;
+    struct route_data_nexthop *rdnh;
+    size_t n_nexthops = 0;
+
+    memset(&rd, 0, sizeof rd);
+    ovs_list_init(&rd.nexthops);
+    if (!nexthop_table_resolve(nhid, &rd)) {
+        return ENOENT;
+    }
+
+    LIST_FOR_EACH (rdnh, nexthop_node, &rd.nexthops) {
```

Cppcheck (2.21.0)

[error] Uninitialized variable

(uninitvar)